src/style.css

```diff
@@ -1,3 +1,35 @@
-body {
-  margin: 0;
+.body {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. If the Roboto font fails to load for some reason, the text will be displayed in the default browser font, which might not look as expected.

Reply:
fixed